srf_generation/velocity_model_generation/q_model_generation/__init__.py

```diff
@@ -0,0 +1,15 @@
+import argparse
+
+
+def load_args():
```

Comment:
It's kinda weird having this in __init__

If it's necessary, a comment explaining why would be good.

Reply:
Huh, that's an artifact of my default template...
Will remove